WordPress/src/main/java/org/wordpress/android/ui/DeepLinkingIntentReceiverActivity.java

```diff
@@ -89,8 +93,18 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
     private void showPost() {
         if (!TextUtils.isEmpty(mBlogId) && !TextUtils.isEmpty(mPostId)) {
             try {
-                ReaderActivityLauncher.showReaderPostDetail(this, mIsFeed, Long.parseLong(mBlogId),
-                        Long.parseLong(mPostId), false);
+                final long blogId = Long.parseLong(mBlogId);
```

Comment:
No need to change this, just wanted to point out you can use `StringUtils.stringToLong()` to do this conversion without needing the exception handler.